ska_sun/sun.py

```diff
@@ -171,6 +194,61 @@ def position_at_jd(jd):
     return ra / dtor, dec / dtor
 
 
+def position_accurate(time, from_chandra=False):
+    """
+    Calculate the sun RA, Dec at the given ``time`` from Earth geocenter or Chandra.
+
+    By default the position is calculated from Earth geocenter.  If ``from_chandra=True``
+    the position is calculated from Chandra using the Chandra predictive ephemeris
+    via the cheta telemetry archive.
+
+    These methods rely on the DE432 ephemeris and functions in ``chandra_aca.planets``.
+    With ``from_chandra=True`` the position should be accurate to within a few arcsec.
+    With ``from_chandra=False`` the position is accurate to within about 0.05 deg.
+
+    Example::
+
+     >>> import ska_sun
+     >>> ska_sun.position_accurate('2008:002:00:01:02')
+     (281.7865848220755, -22.99607130644057)
+
+    :param time: Input time(s) (CxoTimeLike)
+    :param from_chandra: If True compute position from Chandra using ephemeris in cheta
+    :rtype: RA, Dec in decimal degrees (J2000).
+    """
+    func = get_planet_chandra if from_chandra else get_planet_eci
+    eci_sun = func("sun", time)
+    ra, dec = eci_to_radec(eci_sun)
+    return ra, dec
+
+
+def position(time, method=None, **kwargs):
+    """
+    Calculate the sun RA, Dec at the given ``time`` from Earth geocenter or Chandra.
+
+    Example::
```

Comment:
This needs some commentary on the options here and the configuration option here.

Reply:
I stuck some stuff in there that works for me for commentary.